source/frontend/src/features/properties/map/project/ProjectContainer.test.tsx

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { mockLookups } from 'mocks';
+import { mockProjectGetResponse } from 'mocks/mockProjects';
+import { lookupCodesSlice } from 'store/slices/lookupCodes';
+import { render, RenderOptions } from 'utils/test-utils';
+
+import { SideBarContextProvider } from '../context/sidebarContext';
+import ProjectContainer, { IProjectContainerViewProps } from './ProjectContainer';
+
+const mockAxios = new MockAdapter(axios);
```

Comment:
Please do not mock axios. In new code we should mock the providers instead (useProjectProvider)